projects/opentrain/gtfs_pipeline/data_handler/DataHandler.java

```diff
@@ -0,0 +1,29 @@
+package projects.opentrain.gtfs_pipeline.data_handler;
+    
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.List;
+import java.lang.String;
+
+class DataHandler {
+      public static void main(String[] args) throws IOException {
+        System.out.println(args[0]);
+        String outPut = "";
+        Path outfilePath = Paths.get(args[args.length-1]);
+        for (String path : args) {
+            Path curPath = Paths.get(path);
+            List<String> curPathContent = Files.readAllLines(curPath);
+            outPut = outPut + curPath.getFileName() + " lines: ";
+            outPut = outPut + String.valueOf(curPathContent.size()) + "\n";

```

Comment:
Java will automatically convert `curPathContent.size()` to String, you don't need `String.valueOf()`.

Reply:
you're right, fixed